graphene_federation/provides.py

```diff
@@ -1,10 +1,10 @@
-from typing import Any, Union
+from typing import Any, Union, Dict, List
 
 from graphene import Field
 from graphene import Schema
 
 
-def get_provides_parent_types(schema: Schema) -> dict[str, Any]:
```

Comment:
@erikwrede You mean to change the `str` to `String` as well?

Reply:
No, I'm referring to replacing the 3.10 style annotations for things like `list` and `dict` with `typing.List/typing.Dict`, so essentially fixing what was causing your error in your case 🙂

Reply:
https://peps.python.org/pep-0585/ is what is missing from python 3.8 and caused the error for you

Reply:
Sorry if that was misleading, you don't need to prefix everything with typing. - just Dict is Great, like in the previous version!